.github/workflows/linux-cpu-arm64-build.yml

```diff
@@ -41,7 +41,7 @@ jobs:
             --docker-build-args "--build-arg BUILD_UID=$( id -u )" \
             --container-registry onnxruntimebuildcache \
             --repository onnxruntimecpubuild
-          docker run --rm --volume $GITHUB_WORKSPACE:/onnxruntime_src --volume $RUNNER_TEMP:/build -w /build onnxruntimecpubuild bash -c "echo $PATH && /usr/bin/cmake -G Ninja /onnxruntime_src -DCMAKE_BUILD_TYPE=Release -DUSE_CUDA=OFF && ninja"
+          docker run --rm --volume $GITHUB_WORKSPACE:/onnxruntime_src --volume $RUNNER_TEMP:/build -w /build onnxruntimecpubuild bash -c "echo $PATH && /usr/bin/cmake -G Ninja /onnxruntime_src -DBUILD_WHEEL=ON -DCMAKE_BUILD_TYPE=Release -DUSE_CUDA=OFF && ninja"
```

Comment:
By default BUILD_WHEEL is ON in the [CMakeLists.txt](https://github.com/microsoft/onnxruntime-genai/blob/724a24a4c6f5cfb5a88979985ea280046daa7402/CMakeLists.txt#L12).

It should build the wheel without needing to specify BUILD_WHEEL.